steelthread-0/walkthrough.md

```diff
@@ -15,123 +18,234 @@ The installation has already been performed on an OpenShift cluster as per the i
 
 # Setting up a Enmasse queue in your namespace 
 
+- From your namespace click the catalog button on the left hand nav
 - Provision a Enmasse service from the catalog (choose EnMasse standard) name it eval and choose not to bind at this time.
 - Once complete, in the Eval namespace you will find a EnMasse provisioned service.
-- Create a binding to this provisioned service checking each available option.
+- Now create a binding to this provisioned service checking each available option. This will create a secret in the format `enmasse-standard-xxxxx-credentials-yyyyy`
 - Collapse the provisioned service and click on the dashboard link which will show on the right hand side of the EnMasse provisioned service. This link will bring you to the EnMasse dashboard. 
 - When asked to login click the OpenShift button and login with the same user that provisioned the service. In an eval environment this will be ```evals@example.com```
 - Create a new address in the EnMasse console named ```work-queue/requests``` choose the ```queue``` option and the pooled plan.
-- Create a second address in the EnMasse console named ```work-queue/worker-updates``` choose the multicast option and the standard plan
+- Create a second address in the EnMasse console named ```work-queue/worker-updates``` choose the ```queue``` option and the Pooled Queue.
 
-# Setup Enmasse connection in Fuse Ignite
+# Deploy the CRUD REST HTTP booster
 
-Back in your ```evals``` project, You will see a binding and a binding secret as part of the Enmaase provisioned service. There is an option to view this secret. Click on view and click on reveal. Take note of the following values
+This is the HTTP endpoint that will revieve the requests from the Fuse Ignite 
+integreation we create and add new fruits to the inventory.
 
-- messagingHost
-- username
-- password
 
-open the Fuse Ignite route and login in to the Fuse Ignite console using the evals@example.com user 
+Load up the launcher application. This is at ```http://launcher-launcher.<ROUTER_SUFFIX> ``` 
 
-- Select connections and create connection
-- Find the AMQP connection and select it. 
-- Fill in the Connection URI Field with the a URI that is in the following format  ```amqp://<messagingHost>:5672?amqp.saslMechanisms=PLAIN```
-- Fill in the username and password fields
-- Disable Check Certificates
-- Click Validate 
-- Click the blue ```next``` button at the top right
-- Add a connection name ```enmaasse```
-- Click Next / Save
+- Click on launch your project
+- Login when prompted using the ```evals@example.com``` user
+- The create application form will show. Choose ```eval``` as the name (this equates to your namespace)
+- When prompted to select target environment choose : ```Code Locally, Build and Deploy```
+- Click the blue down arrows
```

Comment:
Following a new installation, the next step here in the launcher console is to authorize the cluster?